popup.js

```diff
@@ -0,0 +1,322 @@
+    image: 'images/snap1.png',
+    image2: 'images/portcard.png',
+    description: ` Exploring the future of media in Facebook's first Virtual Reality
+                              app; a place to discover and enjoy 360 photos and videos on Gear
+                              VR.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['FACEBOOK', 'Full Stack Dev', '2015'],
+  },
+  {
+    card: 4,
+    name: 'Multi-Post Stories',
+    name2: 'Uber Navigation',
+    image: 'images/snap3.png',
+    image2: 'images/uber2.png',
+    description: `  A smart assistant to make driving more safe, efficient, and fun by
+                              unlocking your most expensive computer: your car.`,
+    technologies: ['html', 'css', 'javascript'],
+    specifics: ['CANOPY', 'Back End Dev', '2015'],
+    specifics2: ['UBER', 'Lead Developer', '2018'],
+  },
+];
+
+const portfolioSection = document.getElementById('portfolio');
+portfolioSection.innerHTML = projectCards
+  .map(
+    (project, index) => `
+  <section class="card ${index % 2 !== 0 ? 'image-left' : 'image-right'}">
+<div class="content-right">
+          <img class="mobileCard" src="${project.image}" alt="Tonic image" />
+          <img
+            class="desktopCard"
+            src="${project.image2}"
+            alt="Nature image"
+          /></div>
+
+          <div class="content-left">
+          <h2 class="mobileheadline">${project.name}</h2>
+          <div class="">
+            <h2 class="desktopheadline">${project.name2}</h2>
+            <div id="profs" >
+              <h3 id="cannopy">${project.specifics[0]}</h3>
+              <img id="bullet" src="images/bullet.png" alt="" />
+              <h3 class="profList">${project.specifics[1]}</h3>
+              <img  id="bullet2"src="images/bullet.png" alt="" />
+              <h3 class="profList">${project.specifics[2]}</h3>
+            </div>
+            <div id="prof2">
+              <h3  id="cannopy">${project.specifics2[0]}</h3>
+              <img src="images/bullet.png" alt="Counter icon" />
+              <h3 class="">${project.specifics2[1]}</h3>
+              <img src="images/bullet.png" alt="Counter icon" />
+              <h3 class="">${project.specifics2[2]}</h3>
+            </div>
+            <div class="">
+              <p class="tonicm-p">
+              ${project.description}
+              </p>
+              <p class="tonicd-p">
+               ${project.description}
+              </p>
+              <ul class="button">
+                <li>${project.technologies[0]}</li>
+                <li>${project.technologies[1]}</li>
+                <li>${project.technologies[2]}</li>
+              </ul>
+              <ul class="buttonx1">
+                 <li>${project.technologies[0]}</li>
+                <li>${project.technologies[1]}</li>
+                <li>${project.technologies[2]}</li>
+              </ul>
```

Comment:
- [ ] [Optional] You are not following the DRY (Don't repeat yourself) principle because your code is repeated twice to display technologies. I think you are repeating your code twice for mobile and desktop versions, instead of repeating HTML code you can use a media query to target specific devices with proper CSS. 
- [ ] [Optional] I would suggest you use **JavaScript Array map()** to loop through an array to make your code dynamic.